### DISCUSSION TITLE
Breakdown of Commonplace Design Elements

### DISCUSSION THREAD

**Discussion by writingpoli** [knowledgefutures/pubpub#1452] [Show and tell]
I would love a breakdown of the various elements composing [K4's publication, Commonplace](https://commonplace.knowledgefutures.org/). 

Fonts!?

**Comment by gabestein** [comment]
cc @coahearn!

We can't take credit for the design, it was done by @leigler, but I don't see why we couldn't publish some of the style guidelines he put together for us.

**Comment by leigler** [comment]
@writingpoli 
We're using [Old Standard](https://fonts.google.com/specimen/Old+Standard+TT)—inspired by the [LATEX](https://www.latex-project.org/) software's feel.

Each section of the site uses a custom collection of bespoke marbled forms ([Ariel Noltimier Strauss 🙌)](http://arielcassandra.com/) and color—on the homepage they all come together.

We wrote up a piece here if you're interested:
https://commonplace.knowledgefutures.org/pub/y5zqjik4/release/2
